medicines/doc-index-updater/src/pars_upload.rs

```diff
@@ -83,11 +99,10 @@ async fn upload_pars_handler(
 
     let job_id = accept_job(&state_manager).await?.id;
 
-    let _ = tokio::join!(tokio::spawn(queue_pars_upload(
-        job_id,
-        form_data,
-        state_manager.clone(),
-    )),);
+    let span = tracing::info_span!("Queueing PARS upload", job_id = job_id.to_string().as_str());
+    let _enter = span.enter();
+
+    queue_pars_upload(job_id, form_data, state_manager).await?;
```

Comment:
This is a big change in design from before.

Queueing was happening asynchronously so we could give the user the `job_id` quickly, otherwise they'll have to wait for 1 blob storage write and _n_ service bus message pushes.